### ISSUE TITLE
Accessibility code is out of date

### ISSUE THREAD

**Issue by tomgreenfield** [adaptlearning/adapt-contrib-languagePicker#17]
The trimmed down version of `accessibilityView.js` is missing updates added last month, namely https://github.com/adaptlearning/adapt_framework/pull/1142 & https://github.com/adaptlearning/adapt_framework/pull/1194.

**Comment by moloko** [comment]
in working on this I've also noticed that the user's accessibility choice is stored in a cookie rather than using offlineStorage... I reckon I can amend this.

it also does a window.reload when accessibility is switched on and I'm a bit concerned about some of the 'less good' learning managements systems to handle this without freaking out so I'm going to see about changing that too

**Comment by moloko** [comment]
done in https://github.com/adaptlearning/adapt-contrib-languagePicker/commit/19901411badaa35d9d67ed1cdb0c72b605daf41a
